AddShippingZone.tsx

```diff
@@ -1,18 +1,138 @@
-import React from "react";
-import { Button, SafeAreaView, Text, View } from "react-native";
+import React, { useState } from "react";
+import {
+  Linking,
+  Platform,
+  Pressable,
+  SafeAreaView,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from "react-native";
 import { useNavigation } from "@react-navigation/native";
+import CheckBox from "@react-native-community/checkbox";
+import { ToolbarActionButton } from "./ToolbarActionButton";
 
 const AddShippingZone = () => {
   const navigation = useNavigation();
 
+  const [isLimitEnabled, setLimitEnabled] = useState(false);
+
+  React.useEffect(() => {
+    navigation.setOptions({
+      headerRight: () =>
+        ToolbarActionButton({
+          label: "Save",
+          onPress: () => console.log("Toolbar action button pressed"),
+        }),
+    });
+  }, [navigation]);
+
+  const _renderPostCodes = function () {
+    if (isLimitEnabled) {
+      return (
+        <View style={{ marginTop: 10 }}>
+          <Text style={styles.labelText}>Postcodes</Text>
+          <TextInput
+            selectionColor={"black"}
+            style={[styles.textInput, { marginTop: 10 }]}
+            multiline={true}
+            placeholder="List 1 postcode per line"
+          />
+          <Text
+            style={{
+              marginTop: 10,
+              color: "gray",
+            }}
+          >
+            Postcodes containing wildcards (e.g. CB23*) or fully numeric ranges
+            (e.g. 90210...99000) are also supported. Please see the shipping
+            zones{" "}
+            <Text
+              style={{
+                color: "#68a5df",
+                textDecorationLine: "underline",
+              }}
+              onPress={() =>
+                Linking.openURL(
+                  "https://woocommerce.com/document/setting-up-shipping-zones/#section-3"
+                )
+              }
+            >
+              documentation
+            </Text>{" "}
+            for more information.
+          </Text>
+        </View>
+      );
+    } else {
+      return null;
+    }
+  };
+
   return (
     <SafeAreaView>
```

Comment:
```suggestion
    <SafeAreaView style={{ flex: 1 }}>
```

We need the safe area to expand into all the screen or the scroll will look clipped.

![Simulator Screenshot - iPhone 14 Pro - 2023-06-23 at 10 31 40](https://github.com/woocommerce/WooCommerce-Shared/assets/562080/7b95dc09-f8e2-4c0d-b880-eb82d078e9c7)

Reply:
I've added it, but it's strange - I couldn't reproduce it. Maybe because I use simulator for iOS.